.pre-commit-config.yaml

```diff
@@ -113,7 +113,7 @@ repos:
           - requirements.txt
           - --header
           - "# generated from manifests external_dependencies"
-  - repo: https://gitlab.com/PyCQA/flake8
+  - repo: https://github.com/PyCQA/flake8
```

Comment:
Fixed properly in https://github.com/OCA/account-analytic/pull/502. Please rebase.